climada/hazard/tc_tracks.py

```diff
@@ -387,36 +405,52 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
-        self.data = list()
-        fn_nc = SYSTEM_DIR.joinpath(file_name)
-        if not fn_nc.is_file():
+
+        ibtracs_path = SYSTEM_DIR.joinpath(file_name)
+        if not ibtracs_path.is_file():
             try:
                 download_ftp(f'{IBTRACS_URL}/{IBTRACS_FILE}', IBTRACS_FILE)
-                shutil.move(IBTRACS_FILE, fn_nc)
+                shutil.move(IBTRACS_FILE, ibtracs_path)
             except ValueError as err:
                 raise ValueError(
                     f'Error while downloading {IBTRACS_URL}. Try to download it manually and '
-                    f'put the file in {SYSTEM_DIR}') from err
+                    f'put the file in {ibtracs_path}') from err
+
+        ibtracs_ds = xr.open_dataset(ibtracs_path)
+        ibtracs_date = ibtracs_ds.attrs["date_created"]
+        LOGGER.info(f"The IBTrACS data set dates from {ibtracs_date}")
+        if 180 < (np.datetime64('today') - np.datetime64(ibtracs_date)) / np.timedelta64(1, 'D'):
+            LOGGER.warning("The cached IBTrACS data set is older than 180 days. "
+                           "Very likely, a more recent version is available. "
+                           f"Consider manually removing the file {ibtracs_path} and re-running "
+                           "this function, which will download the most recent version of the "
+                           "IBTrACS data set from the official URL.")
 
-        ibtracs_ds = xr.open_dataset(fn_nc)
         match = np.ones(ibtracs_ds.sid.shape[0], dtype=bool)
-        if storm_id:
+        if storm_id is not None:
             if not isinstance(storm_id, list):
                 storm_id = [storm_id]
             match &= ibtracs_ds.sid.isin([i.encode() for i in storm_id])
             if np.count_nonzero(match) == 0:
                 LOGGER.info('No tracks with given IDs %s.', storm_id)
         else:
             year_range = year_range if year_range else (1980, 2018)
-        if year_range:
+        if year_range is not None:
             years = ibtracs_ds.sid.str.slice(0, 4).astype(int)
             match &= (years >= year_range[0]) & (years <= year_range[1])
             if np.count_nonzero(match) == 0:
                 LOGGER.info('No tracks in time range (%s, %s).', *year_range)
-        if basin:
+        if basin is not None:
             match &= (ibtracs_ds.basin == basin.encode()).any(dim='date_time')
             if np.count_nonzero(match) == 0:
                 LOGGER.info('No tracks in basin %s.', basin)
+        if genesis_basin is not None:
+            # Here, we only filter for the basin at *any* eye position. We will filter again later
+            # for the basin of the *first* eye position, but only after restricting to the valid
+            # time steps in the data.
+            match &= (ibtracs_ds.basin == genesis_basin.encode()).any(dim='date_time')
+            if np.count_nonzero(match) == 0:
+                LOGGER.info('No tracks in basin %s.', genesis_basin)
```

Comment:
I know it is not technically 100% correct, but it is more informative to the user to say
```suggestion
                LOGGER.info('No tracks with first valid eye position in genesis basin %s.', genesis_basin)
```

Reply:
Ì modified the output to mention that this refers to the given genesis basin.